exercises/02-react-stateless/src/modules/users/user-types.ts

```diff
@@ -0,0 +1,12 @@
+export interface UserData {
+  firstName: string
+  lastName: string
+}
+
+export interface User extends UserData {
+  id: number
+}
+
+export interface AddUserFunc {
+  (user: UserData): void
+}
```

Comment:
I prefer `type` for functions

```suggestion
export type AddUserFunc = (user: UserData) => void
```